examples/cms-magnolia/templates/components/List.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { EditableArea } from '@magnolia/react-editor';
+
+export default function List(props) {
+	const { items } = props;
+
+	return (
+		<>
+			<div className="hint">[LIST]</div>
+			<ul className="List">
+				<EditableArea
+					content={items}
+					parentTemplateId={props.metadata['mgnl:template']}
+				/>
```

Comment:
Please ensure you have ran Prettier and lint checks.

Reply:
Hey, I have ran Prettier one my side without any issues. The lint indicated deconstruction on props which is done.